src/Shared/OsmObjects.m

```diff
@@ -242,21 +242,14 @@ +(NSInteger)nextUnusedIdentifier
 	__block NSMutableDictionary * merged = [ourTags mutableCopy];
 	[otherTags enumerateKeysAndObjectsUsingBlock:^(NSString * otherKey, NSString * otherValue, BOOL * stop) {
 		NSString * ourValue = merged[otherKey];
-		if (ourValue == nil) {
-			merged[otherKey] = otherValue;
-		} else if ( ![ourValue isEqualToString:otherValue] ) {
+		if ( ![ourValue isEqualToString:otherValue] ) {
 			if ( !allowConflicts ) {
 				if ( IsInterestingTag(otherKey) ) {
 					*stop = YES;
 					merged = nil;
 				}
 			} else {
-				NSArray * a1 = [ourValue componentsSeparatedByString:@";"];
-				NSArray * a2 = [otherValue componentsSeparatedByString:@";"];
-				NSMutableSet * s = [NSMutableSet setWithArray:a1];
-				[s addObjectsFromArray:a2];
-				NSArray * m = [s allObjects];
-				merged[otherKey] = [m componentsJoinedByString:@";"];
+                merged[otherKey] = otherValue;
```

Comment:
The indention looks a bit off in the diff, but in Xcode, everything looks fine. As far as I can tell, the rest of the file is using tabs for indention (even though the _Text Settings_ are set to "Intent Using Spaces").

Ideally, all developers use the same setting. What do you suggest, guys - tabs or spaces? If we decided for the former, it'd probably be helpful to change the _Text Settings_ for all files.

Reply:
I agree there should be standard and it should be documented or shared somewhere. 

@wtimme do you believe this is blocking the PR?

Reply:
No, I don't. I only thought we should have a general discussion about this - not something that is immediately related to your pull request.

I think one option could be to use an automatic code formatter (cp. #201). That would make sure every contributor adheres to the standard that we agreed upon - even removing the need to document it in text, since it was then documented as part of a formatter configuration file.

Reply:
I agree. I'll resolve this conversation and we can move the the discussion to #201